pandas/sparse/series.py

```diff
@@ -29,12 +29,14 @@
 
 from pandas.util.decorators import Appender
 
+from pandas.sparse.scipy_sparse import sparse_series_to_coo, coo_to_sparse_series
```

Comment:
I've moved "import scipy" inside the method. If you run without scipy you get an import error when you call the to_coo function ... I think this is probably ok behaviour.